isis/tests/Fixtures.cpp

```diff
@@ -21,6 +23,14 @@ namespace Isis {
     testCube = new Cube();
     testCube->fromIsd(tempDir.path() + "/default.cub", label, isd, "rw");
 
+    LineManager line(*testCube);
+    for(line.begin(); !line.end(); line++) {
+      for(int i = 0; i < line.size(); i++) {
+        line[i] = (double) 1;
+      }
+      testCube->write(line);
+    }
```

Comment:
Maybe I am missing something, but it seems like you are adding lines on the fly here, but the fixture on disk also has binary data?

Reply:
The LineManager loops over the lines that are already in the cube.

Reply:
sounds reasonable enough